docs/explanation/authentication.rst

```diff
@@ -3,7 +3,8 @@ Authentication and Authorisation
 
 Authentication requires a client_id and a secret_key. These credentials can be
 obtained by contacting the server administrator with the queues you want priority
-access for as well as the maximum priority level to set for each queue. The
-expectation is that these credentials are shared between users on a team. 
+access for, the maximum priority level to set for each queue, and any restricted
+queues that you need access to. The expectation is that these credentials are
+shared between users on a team.
```

Comment:
I think you can eliminate the last sentence about "The expectation is that these credentials are shared between users on a team." That's a bit of a detail with how we're using it, but in reality it could be a single user, a bot account, or whatever. There's no need to narrow the scope of how it's used here.